src/installer-source/KolibriSetupScript.iss

```diff
@@ -537,12 +537,34 @@ end;
 function GetPipDir(Value: string): String;
 begin
     result := ExtractFileDir(GetPipPath());
+end;
+
+{REF: https://stackoverflow.com/questions/3304463/how-do-i-modify-the-path-environment-variable-when-running-an-inno-setup-install}
+function EnvAddPath(param: string): boolean;
+var
+  Paths: string;
+begin
+  { Retrieve current path (use empty string if entry not exists) }
+    if not RegQueryStringValue(HKEY_LOCAL_MACHINE, 'System\CurrentControlSet\Control\Session Manager\Environment', 'Path', Paths)
+    then Paths := '';
+
+    { Skip if string already found in path }
+    if Pos(';' + Uppercase(param) + ';', ';' + Uppercase(Paths) + ';') > 0 then exit;
+
+    { App string to the end of the path variable }
+    Paths := Paths + ';'+ param +';'

```

Comment:
Maybe we should prepend our new path here to make sure it executes first in case there's an old path declaration?